commands/management/add_mysql.go

```diff
@@ -254,6 +273,7 @@ func init() {
 	AddMySQLC.Flag("username", "MySQL username").Default("root").StringVar(&AddMySQL.Username)
 	AddMySQLC.Flag("password", "MySQL password").StringVar(&AddMySQL.Password)
 	AddMySQLC.Flag("agent-password", "Custom password for /metrics endpoint").StringVar(&AddMySQL.AgentPassword)
+	AddMySQLC.Flag("credentials", "Credentials provider").ExistingFileVar(&AddMySQL.Credentials)
```

Comment:
```suggestion
	AddMySQLC.Flag("credentials-file", "Path to credentials").ExistingFileVar(&AddMySQL.Credentials)
```

Reply:
This was left as a placeholder whilst discussing accepting executables as well as files

Reply:
I think that `credentials-src` or `credentials-source` might be more accurate. At least in my mind, using `-file` would imply the reading the file vs the possible execution and thus reading output

Reply:
Also, `credentials-s{ource,rc}` would easily extend to support HTTP requests

Reply:
@BupycHuk any input based upon these comments? I have left it as `credentials` for now so that they can just get replaced in one go

Reply:
I don't think that there will be possibility to execute files, it's security risk. So let's make it `-file`

Reply:
@BupycHuk see [my comment](https://jira.percona.com/browse/PMM-9590?focusedCommentId=295545&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-295545) on the ticket about this. I think that there are safe enough ways to do this once the plugin architecture is available, as the user could register a provider through that, allowing for signature comparison ahead of execution.